target-bin/bootstrap-fixup.in

```diff
@@ -16,13 +16,15 @@ elif [ $DISTRIB_NAME = "Debian" ]; then
   # grub-legacy conflicts grub-pc dependencies
   # No grub-legacy on Ubuntu, just on Debian
   # Work around bcron-run conflict due to cron being removed
-  apt-get purge -y grub-legacy bcron-run &> /dev/null
+  # Needed for KVM, but apparently errors out for LXC, so the true ignores the error
+  apt-get purge -y grub-legacy bcron-run &> /dev/null || true
 fi
 echo '127.0.1.1 gitian' >> /etc/hosts
 
 # If LXC
 if grep /lxc/gitian /proc/1/cgroup > /dev/null; then
-  adduser --disabled-password --gecos ubuntu --quiet ubuntu || true
+  adduser --disabled-password --gecos ${DISTRIB_NAME,,} --quiet ${DISTRIB_NAME,,} || true
+  adduser ${DISTRIB_NAME,,} sudo
```

Comment:
I understand.  So the plan is to have sudo enabled through the build descriptor + command line flag (search for sudo in gbuild for details).  I'll disable the blanket sudo permission in the vmbuilder path.

This is for security reasons.  Let me know if any objections.